www/pfSense-pkg-vHosts/files/usr/local/www/vhosts_certs.php

```diff
@@ -0,0 +1,465 @@
+<?php
```

Comment:
Your point is well taken. The first iteration of vHosts did use Cert Mgr. There were a couple of things I wanted that Cert Mgr did not provide and I did not want to try and modify that code. Also, I don't know how Cert Mgr determines if a cert is in use but if there is not some generic mechanism for this, Cert Mgr would have to know about vHosts which would violate the orthogonal nature of packages. In fact, I actually found a reference to the original vHosts package that still resides in the base code. I considered that I might modify Cert Mgr in the future.